spec/factories/user.rb

```diff
@@ -28,5 +28,10 @@
     trait :mfa_enabled do
       otp_required_for_login { true }
     end
+
+    trait :no_mobile_number do
+      mobile_number { nil }
+      mobile_number_confirmed_at { nil }
```

Comment:
here we're specifying that `mobile_number_confirmed_at` is nil, but we're not setting it for all other users, so its not really overriding anything. I think our default user factory should have this set to a date in the past.

Reply:
That is absolutely true but we're describing a known state we *want* as a result of the trait and indicating that these fields are to be taken together; this is largely because people can add things they believe to be unrelated in other FactoryBot places and break tests. It's as much a cue for the reader as a programmatic "must set".

Reply:
I'm not suggesting removing this trait, I agree it should exist, and I like it. I'm suggesting that the default user factory should have a mobile number confirmed, as that's our common case.

Reply:
Oh I see. In that case we'll add.